spec/Section 7 -- Response.md

```diff
@@ -244,6 +264,77 @@ discouraged.
 }
 ```
 
+### Incremental
+
+The `incremental` entry in the response is a non-empty list of Defer or Stream
```

Comment:
I think it would be good if we added an example of the json structure like above.

Reply:
```json
{
        data: {
          first: {},
          second: { theNumber: 2 },
        },
        hasNext: true,
      },
      {
        incremental: [
          {
            label: 'defer-label',
            path: ['first'],
            data: {
              promiseToGetTheNumber: 2,
            },
          },
        ],
        hasNext: false,
      }
      ```

Reply:
is it "data" or "items" key (see below)?

Reply:
I added a full example to this section